docs/examples/DataSet/Dataset Context Manager.ipynb

```diff
@@ -6801,7 +6810,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Unlike the data measured on a grid above all the measured data points have an unique combination of the two dependent parameters. When exporting to XArray NaN's will therefore replace all the missing combinations of `dac_ch1` and `dac_ch2` and the data is unlikely to be useful in this format. "
+    "Unlike the data measured on a grid, all the measured data points have an unique combination of the two dependent parameters. When exporting to XArray NaN's will therefore replace all the missing combinations of `dac_ch1` and `dac_ch2` and the data is unlikely to be useful in this format. "
```

Comment:
Maybe:
Unlike the data measured above, which lies on a grid, here, all the ...
(not sure if this was the intention)